tests/test_conflicting_formatters.py

```diff
@@ -0,0 +1,72 @@
+import tempfile
+from typing import Generator, List, Type
```

Comment:
```suggestion
from __future__ import annotations

from collections.abc import Generator

import tempfile
```

This allows you to use `list[]` and `type[]`.